2024/README.md

```diff
@@ -34,7 +34,7 @@ p { text-align: justify; }
 
 <br>
 
- In conjunction with:  
+ In conjunction with:
```

Comment:
Keep for markdown "[soft-break newline](http://eherrera.net/markdowntutorial/tutorial/paragraphs.html)"
```suggestion
 In conjunction with:  
```